doc/sphinx/source/notes_7_0.rst

```diff
@@ -2,10 +2,27 @@
 Release Notes for PoCL 6.1
 **************************
 
+* removed support for building tests & examples with OpenCL < 3.0
```

Comment:
This doesn't look good as a highliner :) We used to have "deprecation notes" or similar section in the end. Should be there?

Reply:
it's not a "highliner", see below